domain/consensus/datastructures/pruningstore/imported_pruning_point.go

```diff
@@ -0,0 +1,211 @@
+package pruningstore
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/kaspanet/kaspad/domain/consensus/database/serialization"
+	"github.com/kaspanet/kaspad/domain/consensus/model"
+	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/dbkeys"
+)
+
+var importedPruningPointUTXOsBucket = dbkeys.MakeBucket([]byte("imported-pruning-point-utxos"))
+var importedPruningPointMultiset = dbkeys.MakeBucket().Key([]byte("imported-pruning-point-multiset"))
+
+func (ps *pruningStore) ClearImportedPruningPointUTXOs(dbTx model.DBTransaction) error {
+	cursor, err := dbTx.Cursor(importedPruningPointUTXOsBucket)
+	if err != nil {
+		return err
+	}
+
+	for cursor.Next() {
+		key, err := cursor.Key()
+		if err != nil {
+			return err
+		}
+		err = dbTx.Delete(key)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
This will create a batch the size of the keys. Not sure what to do about this, though.
Dealing with this without a batch will probably require another flag, and I'm not sure it's worth it.

Reply:
why can't we have a single flag from start to end?